akka-docs/rst/java/code/docs/http/javadsl/server/directives/RespondWithDirectivesExamplesTest.java

```diff
@@ -0,0 +1,131 @@
+package docs.http.javadsl.server.directives;
```

Comment:
Should have license header (like all files, just copy the comment from the top - thanks!)